starfish/intensity_table/intensity_table.py

```diff
@@ -365,10 +392,10 @@ def from_image_stack(
         assert crop_y * 2 < image_stack.shape['y']
         assert crop_x * 2 < image_stack.shape['x']
 
-        zmin = crop_z
+        zmin = image_stack.axis_labels(Axes.ZPLANE)[crop_z]
         ymin = crop_y
         xmin = crop_x
-        zmax = image_stack.shape['z'] - crop_z
+        zmax = image_stack.axis_labels(Axes.ZPLANE)[-crop_z - 1]
```

Comment:
-crop_z - 1 this is weird, what value is this?

Reply:
So this is to allow for cropping from the maximum zlayer.  crop_z=0 => zmax=-1 (i.e., last plane).  crop_z=1 => zmax=-2 (i.e., second to last plane).